interface/main/calendar/add_edit_event.php

```diff
@@ -1428,6 +1549,33 @@ function find_available(extra) {
   </td>
  </tr>
 
+    <style>
+        #days_every_week_row input[type="checkbox"]{float:right;}
+        #days_every_week_row div{display: inline-block; text-align: center; width: 12%;}
+        #days_every_week_row div input{width: 100%;}
+    </style>
+
+<tr id="days_every_week_row">
+    <td></td>
+    <td></td>
+    <td><input  type='checkbox' name='days_every_week' onclick='set_days_every_week()' <?php if (isDaysEveryWeek($repeats)) echo " checked" ?>/></td>
+    <td id="days_label"><?php echo xlt('Days Of Week:'); ?></td>
```

Comment:
remove colon and whitespace from the constant:
``````
<td id="days_label"><?php echo xlt('Days Of Week:'); ?></td>